packages/next/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -130,22 +130,6 @@ export default class BuildManifestPlugin {
           assetMap.pages['/'] = assetMap.pages['/index']
         }
 
-        // Add the runtime build manifest file (generated later in this file)
-        // as a dependency for the app. If the flag is false, the file won't be
-        // downloaded by the client.
-        if (this.clientManifest) {
-          assetMap.pages['/_app'].push(
```

Comment:
Should we still inject this to the `assetMap` and continue filtering it out later?